parent/pom.xml

```diff
@@ -77,7 +77,8 @@
     <version.msgpack>0.9.9</version.msgpack>
     <version.netty>4.1.117.Final</version.netty>
     <version.objenesis>3.4</version.objenesis>
-    <version.opensearch>2.14.0</version.opensearch>
+    <version.opensearch-java>2.19.0</version.opensearch-java>
+    <version.opensearch>2.18.0</version.opensearch>
```

Comment:
`opensearch-java` has published 2.19.0 with the fix, but `opensearch-rest-client` has not released 2.19.0 yet.